apps/playground-lit/src/app/app.element.ts

```diff
@@ -17,18 +17,24 @@ export class AppElement extends LitElement {
 
     render(): TemplateResult {
         return html`
-            <a is="vl-link" id="btn-acties">Acties</a>
-            <vl-popover data-vl-for="btn-acties" data-vl-placement="bottom-start">
-                <ul is="vl-link-list">
-                    <li is="vl-link-list-item">
-                        <a is="vl-link">Voeg gebruiker toe.</a>
-                    </li>
-                    <li is="vl-link-list-item">
-                        <a is="vl-link">Voeg adres toe.</a>
-                    </li>
-                </ul>
-            </vl-popover
-
+            <main>
+                <a is="vl-link" id="btn-acties">Acties</a>
+                <vl-popover for="btn-acties" placement="bottom-start">
+                    <vl-popover-action-list
+                        @click=${(event: CustomEvent) => {
+                            const actionElement = event.target as VlPopoverActionComponent;
+                            if (actionElement instanceof VlPopoverActionComponent) {
```

Comment:
is dit nodig? (aangezien je al cast?)
(kan zijn dat ik hier internals van js mis in m'n hoofd)

Reply:
ja zonder cast wordt het gezien als `any`, heb het wel aangepast naar `Event`